### PR TITLE
updated readMe

### PR DESCRIPTION
**Pull request by shreya-mishra**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,41 +24,59 @@ Feel free to download our RDS app by clicking on ➡️ [RDS App](https://drive.
...
 - First set up with React native environment on your local machine. You can refer the [Environment Setup Link](https://reactnative.dev/docs/environment-setup)
 - If you have done the first point then congrats 🎉now you are good to go !
 - Now clone the mobile app Repo with [Github Repo Link](https://github.com/Real-Dev-Squad/mobile-app)
-- Create a new file: config/config.js. copy content from config/config.sample.js into config/config.js.
-- Register the application for GitHub OAuth to get the clientId and clientSecret. Add the callback URL as x-realdevsquad-rdsapp://oauth2/authorize
-- Replace the clientId and clientSecret in `config/config.js` file with the values from the github oauth app
+- Run the command - `Yarn`.
 - Start contributing.
 
 ## Contributuion Guide
```
Comment (sakshambhatt) [inline README.md]: Please correct the typo: contribution

File: README.md [hunk 2]
```diff
@@ -24,41 +24,59 @@ Feel free to download our RDS app by clicking on ➡️ [RDS App](https://drive.
...
 
-- If you are new to React Native please go through react-native documentation and setup the installation setup first.
-- Clone the repo and go through the code base.
-- Make sure you have node installed on your computer.
-- Go through the issues and comment on issue which you find interesting.
-- Discuss, contribute and raise the PR.
+- You must have all the pre-requisites which is mentioned above.
```
Comment (sakshambhatt) [inline README.md]: Grammar: which *are* mentioned

File: README.md [hunk 3]
```diff
@@ -24,41 +24,59 @@ Feel free to download our RDS app by clicking on ➡️ [RDS App](https://drive.
...
+- Go through the issues and comment on the same which you like.
```
Comment (sakshambhatt) [inline README.md]: Grammar: comment on the one which you like*

File: README.md [hunk 4]
```diff
@@ -24,41 +24,59 @@ Feel free to download our RDS app by clicking on ➡️ [RDS App](https://drive.
...
+- When the issue is assigned to you then just follow all the local App setup steps and congrats you are good to do😊
```
Comment (sakshambhatt) [inline README.md]: Grammar: please remove congrats

File: README.md [hunk 5]
```diff
@@ -24,41 +24,59 @@ Feel free to download our RDS app by clicking on ➡️ [RDS App](https://drive.
...
+- Before pushing the code make sure you follow the below 👇steps
+
+  - Make sure you run the app on your local machine. It should not give any errors.
+  - Run `yarn run format-fix`
+  - Run `yarn run lint` for fix linting errors.
```
Comment (sakshambhatt) [inline README.md]: to fix linting errors*

File: README.md (hunk 4 above)
Comment (sakshambhatt) [reply README.md]: Grammar: When an issue is assigned*

File: README.md [hunk 6]
```diff
@@ -24,41 +24,59 @@ Feel free to download our RDS app by clicking on ➡️ [RDS App](https://drive.
...
+  - Remove unused comments and console logs.
+  - Run `yarn run test`.
+  - then Run `your git commands` to push your code.
+  - Make sure give the proper commit message.
```
Comment (sakshambhatt) [inline README.md]: Grammar: Make sure to give a proper commit message*

File: README.md [hunk 7]
```diff
@@ -24,41 +24,59 @@ Feel free to download our RDS app by clicking on ➡️ [RDS App](https://drive.
...
+- `git push origin your PR name`
+
+
 **Running/Development**
 
-- `npx react-native start` - to start Metro, the JavaScript bundler that ships with React Native.
+- `yarn react-native start` - to start Metro, the JavaScript bundler that ships with React Native.
 - Let Metro Bundler run in its own terminal. Open a new terminal inside your React Native project folder. Run the following:
-- `npx react-native run-android` - To start the emulator.
+- `yarn react-native run-android` - To start the emulator.
```
Comment (sakshambhatt) [inline README.md:82]: I appreciate that things are updated as per the current changes. 
The steps mentioned for committing, and other pre-requisites should be helpful to someone new.
Rest of the things are fine :+1:

**Review by sakshambhatt — CHANGES_REQUESTED** [review]

**Review by sakshambhatt — APPROVED** [review]
LGTM!! :rocket:

**Review by sakshambhatt — APPROVED** [review]

**Comment by vinayak-trivedi** [comment]
Looks good to me

**Review by vinayak-trivedi — APPROVED** [review]
Looks good to me

**Pull request merged**